run.sh

```diff
@@ -104,21 +112,29 @@ bench() {
     if [[ "$TARGET_ENV" = "development" ]] && $DEPLOY_KIND_OBSERVATORIUM; then
         echo "Deploying observatorium dev manifests"
         deploy_observatorium
-    fi
 
-    echo -e "\nForward ports to loki deployments"
-    forward_ports
+        wait_for_deployments
 
-    echo -e "\nSet prometheus relabel regex"
-    set_prometheus_relabel_regex
+        echo -e "\nForward ports to loki deployments"
+        forward_ports
 
-    echo -e "\nScrape metrics from Loki deployments"
-    scrape_loki_metrics
+        echo -e "\nSet prometheus relabel regex"
+        set_prometheus_relabel_regex
+
+        echo -e "\nScrape metrics from Loki deployments"
+        scrape_loki_metrics
+    else
+        secret=$($KUBECTL -n openshift-user-workload-monitoring get secret | grep prometheus-user-workload-token | head -n 1 | awk '{print $1 }')
+        PROMETHEUS_URL="https://$($KUBECTL -n openshift-monitoring get route thanos-querier -o json | jq -r '.spec.host')"
+        PROMETHEUS_TOKEN=$($KUBECTL -n openshift-user-workload-monitoring get secret $secret -o json | jq -r '.data.token' | base64 -d)
+
+        wait_for_deployments
+    fi
```

Comment:
This should be an optional addition to `loki-benchmarks` and in general non-OCP specific.